CHANGELOG.rst

```diff
@@ -6,7 +6,7 @@ This document describes changes between each past release.
 12.1.0 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Update Kinto OpenID plugin to redirect with a base64 JSON encoded token. (#1988)
```

Comment:
We should add a note that this is a breaking change IMO. At least mention the minimum required version of kinto-admin :)